src/create-virtual-symlink.ts

```diff
@@ -51,27 +50,20 @@ export async function createVirtualSymlink(
       } catch (e) {}
     }
     process.stdin.resume();
-    process.on('exit', () => exitHandler(originalPackageJson));
-    process.on('SIGINT', () => exitHandler(originalPackageJson));
-    process.on('SIGUSR1', () => exitHandler(originalPackageJson));
-    process.on('SIGUSR2', () => exitHandler(originalPackageJson));
-    process.on('uncaughtException', () => exitHandler(originalPackageJson));
+    process.on('exit', () => exitHandler(originalPackageJson, false));
+    process.on('SIGINT', () => exitHandler(originalPackageJson, false));
+    process.on('SIGUSR1', () => exitHandler(originalPackageJson, false));
+    process.on('SIGUSR2', () => exitHandler(originalPackageJson, false));
+    process.on('uncaughtException', () =>
+      exitHandler(originalPackageJson, false),
+    );
     await modifyJson(packageJson, dependencies, outFolder, outFolderName);
   }
-  await Worker({
-    command: isWin ? 'cmd' : 'npx',
-    args: [
-      ...(isWin ? ['/c', 'npx'] : []),
-      runner,
-      ...process.argv
-        .slice(2)
-        .filter(
-          a =>
-            a !== Tasks.LEAVE_CHANGES &&
-            a !== Tasks.REVERT &&
-            a !== Tasks.BUILD,
-        ),
-    ],
-  });
-  exitHandler(originalPackageJson);
+  try {
+    await runCommand(runner, process.argv);
+    exitHandler(originalPackageJson, true);
```

Comment:
Missing `await` here since `exitHandler` is `async` this can lead to missed error inside `try` and `catch` scope.

I will fix it after the merge so don't worry! :)

Reply:
You're totally right, I missed the async there. Thanks!